front-end/damaFirstPage.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang='en'>
+  <head>
+    <meta charset='UTF-8'/>
+    <title>VivaLaDama</title>
+    <link rel='stylesheet' href='stylesFirstPage.css'/>
+  </head>
+  <body>
+    <div id='title'><h1>VIVA LA DAMA !</h1></div>
+    <div id='menu'>
+      <form action="../DamAliasLab/front-end/damaMainPage.html">
+        <button type="submit">New Game</button>  
+      </form>
+      <form>
+        <div id='resumeButton' onclick="viewGamesTable()">Resume Game..</div>  
+      </form>
+    </div>
+    <div id='container-tab'>
+      <table id='table-games' cellspacing='0'>
+        <tr id='heading'>
+          <th class='first-column'>ID</td>
+          <th class='second-column'>Players</td>
+          <th class='third-column'>Points</td>
+        </tr>
+        <tr class='rows'>
+          <td>ID1</td>
+          <td>Player1 - Player2</td>
+          <td>500-200</td>
+        </tr>
+        <tr class='rows'>
+          <td>ID2</td>
+          <td>Player3 - Player4</td>
+          <td>100-90</td>
+        </tr>
+        <tr class='rows'>
+          <td>ID3</td>
+          <td>Player5 - Player6</td>
+          <td>10-50</td>
+        </tr>
+      </table>
+    </div>
+    
+  </body>
+  <script>
+    function viewGamesTable(){
+      elem=document.getElementById('container-tab');
+      elem.style.display='flex';
+    }
+  </script>
```

Comment:
Perchè questo script è in fondo alla pagina? Se vuoi includere i file dopo che l'html sia caricato ti consiglio di vedere [](https://javascript.info/script-async-defer)